client/app/src/state/reducer.js

```diff
@@ -0,0 +1,27 @@
+import actionTypes from "./actionTypes";
+
+const INITIAL_STATE = {
+  lastKnownScreen: "Welcome",
```

Comment:
Is this going to be used for anything?

Reply:
I don't think it will be used for anything, given that the current stageId is provided on connection to the websocket. I'll remove it.

Reply:
However, it may make sense to store the current stage id, depending on whether the stage-specific attributes of previous stages will need to be kept for some reason (see below).